docker/consul-template-bookbrainz.conf

```diff
@@ -0,0 +1,11 @@
+template {
+    source = "/home/bookbrainz/bookbrainz-site/config.json.ctmpl"
+    destination = "/home/bookbrainz/bookbrainz-site/config/config.json"
+}
+
+exec {
+    command = "/usr/local/bin/bookbrainz"
+    reload_signal = ""
```

Comment:
Setting this to nil means it'll restart the process every time the template changes, causing a small amount of downtime. Which you might be aware of, but just in case :)

Reply:
If there's a change, that's the only way we can get up-to-date. I'd like to be able to respond to SIGHUP, but that will take some work.